pkg/agent/openflow/network_policy_test.go

```diff
@@ -546,3 +594,47 @@ func prepareClient(ctrl *gomock.Controller) *client {
 	c.ofEntryOperations = m
 	return c
 }
+
+func TestParseMetricFlow(t *testing.T) {
+	for name, tc := range map[string]struct {
+		flow   string
+		rule   uint32
+		metric types.RuleMetric
+	}{
+		"Drop flow": {
+			flow: "table=101, n_packets=9, n_bytes=666, priority=200,ip,reg0=0x100000/0x100000,reg3=0x5 actions=drop",
+			rule: 5,
+			metric: types.RuleMetric{
+				Bytes:    666,
+				Packets:  9,
+				Sessions: 9,
+			},
+		},
+		"New Pass flow": {
```

Comment:
maybe you should use "Allow" instead of "Pass" for consistency with the rest of the code / comments?

Reply:
Updated.